src/bin/simple_test/main.rs

```diff
@@ -101,8 +101,8 @@ fn simple_test(iface_name: &str) -> i32 {
         Ok(_) => (),
     };
 
-    let try = 40;
-    for _ in 0..try {
+    let r#try = 40;
```

Comment:
Why should we use `r#try` name using hash sign? As far as I know, `retry` is not a keyword in Rust.

Reply:
This change was made automatically by `cargo fix --edition`.
Now I removed the variable entirely :)